### PR TITLE
Fix Travis CI

### PR DESCRIPTION
**Pull request by qmfrederik**
The code in this repository is very old, and this PR makes a couple of changes to make CI work on new versions of Xcode:

- Use `xcodebuild` instead of `xctool` to build the projects
- Upgrade `.xib` files to the new file format
- Target iOS 8 or later (up from iOS 5)
- Link with OpenSSL from Homebrew

### CODE REVIEW

**Comment by qmfrederik** [comment]
There's a lot of unused code in this repository. For the scope of WebDriverAgent, you may want to consider removing the following pieces of code:

- WebDAV support
- WebSocket support
- SSL support
- All examples but iPhoneHTTPServer

File: Samples/SecureHTTPServer/SecureHTTPServer.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -441,6 +441,7 @@
 			buildSettings = {
 				GCC_PREFIX_HEADER = SecureHTTPServer_Prefix.pch;
 				INFOPLIST_FILE = Info.plist;
+				LIBRARY_SEARCH_PATHS = "/usr/local/opt/openssl@1.1/lib";
```
Comment (mykola-mokhnach) [inline Samples/SecureHTTPServer/SecureHTTPServer.xcodeproj/project.pbxproj:444]: Should this be mentioned somewhere in readme?

**Comment by mykola-mokhnach** [comment]
Thanks for updating this @qmfrederik 

Regarding modules support: I suppose SSL/TLS and web sockets might be necessary. The other stuff is for sure not needed.

**Comment by mykola-mokhnach** [comment]
Also, keep in mind that the module supports TvOS arch as well

**Review by mykola-mokhnach — APPROVED** [review]

**Review by KazuCocoa — APPROVED** [review]
awesome

**Pull request merged**